Source/ACE.Server/WorldObjects/Monster_Awareness.cs

```diff
@@ -80,6 +80,7 @@ public virtual void HandleFindTarget()
         {
             return;
 
+            /*
```

Comment:
If you don't mind my asking, would you mind sharing what error you resolved by commenting out this code block? I believe this is a fairly critical piece of the monster 'think' logic.

Reply:
there's a `return;` line just above the commenting so nothing is getting past that making this code useless. I wanted to clean up the existing warnings that have been lingering for months

Reply:
Got it. I'm sorry. I didn't 'review' thoroughly enough to see that! Thank you for taking the time to clean up those lingering warnings at compile time!

Reply:
Not a problem!

Reply:
This is WIP monster aggro code... it's not being used yet, so it just returns immediately

Commenting it out for now is a good idea